src/main/resources/org/jenkinsci/plugins/configfiles/maven/GlobalMavenSettingsConfig/edit-config.jelly

```diff
@@ -36,9 +36,9 @@ div.credentialssection {
 	   		<p>
 	   		 ${%description}
 	   		</p>
-	   		
-            <input type="hidden" name="config.id" value="${config.id}" />
-            
+            <f:entry title="${%ID}" >
+                <f:textbox readonly="readonly" name="config.id" value="${config.id}" />
+            </f:entry>
             <f:entry title="${%Name}">
                 <f:textbox name="config.name" value="${config.name}" />
             </f:entry>
```

Comment:
Might be a good time to factor out the common parts here into a shared `*.jelly`.